common/STcommon.py

```diff
@@ -31,12 +67,30 @@ def parseArgs():
 	'''Parses args using the argparse lib'''
 	parser = argparse.ArgumentParser(description='Location logging server')
 
+	parser.add_argument('-a', '--address', metavar='ADDRESS')
+	parser.add_argument('-c', '--configure', action='store_true')
 	parser.add_argument('-g', '--generate-keys', metavar='PATH', type=str)
+	parser.add_argument('-L', '--logfile', metavar='LOG_FILE')
```

Comment:
I believe uppercase `L` is preferable to lowercase `l` to avoid confusion with uppercase `I`.